release-notes/release-notes58.md

```diff
@@ -188,14 +188,28 @@ This section describes issues that users of {{site.productname}} 5.8 may encount
 
 **Outline**
 
-- [](#)
+- [Unexpected behavior when moving the caret using the PageDown and End keys](#unexpectedbehaviorwhenmovingthecaretusingthepagedownandendkeys)
+- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)
+- [Content pasted from Google Docs cannot be outdented](#contentpastedfromgoogledocscannotbeoutdented)
 
-###
+### Unexpected behavior when moving the caret using the PageDown and End keys
 
-**Issue**: This issue affects ....
+**Issue**: This issue affects moving the caret over links using the PageDown or End keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.
```

Comment:
We generally **bold** the names of keyboard keys elsewhere in the docs.
```suggestion
**Issue**: This issue affects moving the caret over links using the **PageDown** or **End** keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.
```

Reply:
Don't we normally put them in code blocks, not make them bold?

Reply:
Looking at the keyboard shortcuts page, backticks

Reply:
```suggestion
**Issue**: This issue affects moving the caret over links using the `Page Down` or `End` keys. If the caret is placed adjacent to a link and the key is pressed, the caret will move inside the link rather than past it.
```